libkbfs/key_manager.go

```diff
@@ -311,22 +311,24 @@ func (km *KeyManagerStandard) unmaskTLFCryptKey(ctx context.Context, serverHalfI
 	return tlfCryptKey, nil
 }
 
-func (km *KeyManagerStandard) updateKeyBundle(ctx context.Context,
-	md *RootMetadata, keyGen KeyGen,
-	wKeys, rKeys map[keybase1.UID][]kbfscrypto.CryptPublicKey,
+func (km *KeyManagerStandard) updateKeyGeneration(ctx context.Context,
+	md *RootMetadata, keyGen KeyGen, wKeys, rKeys UserDevicePublicKeys,
 	ePubKey kbfscrypto.TLFEphemeralPublicKey,
 	ePrivKey kbfscrypto.TLFEphemeralPrivateKey,
 	tlfCryptKey kbfscrypto.TLFCryptKey) error {
 
-	newServerKeys, err := md.fillInDevices(km.config.Crypto(),
+	serverHalves, err := md.updateKeyGeneration(km.config.Crypto(),
 		keyGen, wKeys, rKeys, ePubKey, ePrivKey, tlfCryptKey)
 	if err != nil {
 		return err
 	}
 
 	// Push new keys to the key server.
+	//
+	// TODO: Should accumulate this and push them all at once
+	// right before the MD push.
```

Comment:
For MDv2, rekeying has to update all key generations, so that generates some number of server halves per generation, so 'accumulate' refers to the multiple generations. I guess it's less important for MDv3. Clarified comment.